authentication.md

```diff
@@ -57,8 +57,34 @@ When correctly authenticated, the response will return 200 and in this example s
 
 ## Refreshing a token
 
-The token will expire after one week. You have either to log in again or use the refresh token.  
-To be tested and documented, ticket [TC-4734](https://tradecloud.atlassian.net/browse/TC-4734)
+Access token will expire after 1 hour, refresh token after 24 hours.
+If access token expires you have either to log in again or use the refresh token.
+If refresh token expires you have to log in again.
+
+To refresh token:
+Make http call to `/refresh` endpoint using Refresh-Token header:
+```text
+// Example request method and URI
+GET https://api.accp.tradecloud1.com/v2/authentication/_refresh
+// Request headers:
+Refresh-Token: <Refresh-Token>
+```
+
+When refresh token is valid, the response will return 200 and contain new token pairs, otherwise you will get 401 "Not authenticated".
+The old ones are discarded and cannot be used any more.
```

Comment:
Suggested: "Access tokens and refresh tokens cannot be used once they are expired or a new access token is generated."

Reply:
@roy-tc There was some misunderstanding how tokens can be used when new pair is generated. Actually, only refresh token can not be used once new refresh token is generated . But access token can. And this is a problem with JWT tokens in general. That's why logout (aka session invalidation in terms of `akka-http-session`) doesn't cause that we can't use access token anymore.

https://github.com/softwaremill/akka-http-session-faq#even-when-a-session-is-invalidated-by-invalidatesession-is-it-still-available